shotgun_api3/shotgun.py

```diff
@@ -598,6 +599,7 @@ def __init__(self,
         self.config.convert_datetimes_to_utc = convert_datetimes_to_utc
         self.config.no_ssl_validation = NO_SSL_VALIDATION
         self.config.raw_http_proxy = http_proxy
+        self.config.localized = False
```

Comment:
i don't think we need this line - it's already initialized in the constructor for `_Config`, right? If we do need it, please explain the reason why with a comment!

Reply:
That's correct, I removed the line